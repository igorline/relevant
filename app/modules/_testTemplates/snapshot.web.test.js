import React from 'react';
import renderer from 'react-test-renderer';
import AvatarBoxWeb from 'modules/user/web/avatarbox.component';
import 'jest-styled-components';
import { MemoryRouter } from 'react-router-dom';

const user = { _id: 123, handle: 'handle', name: 'Name', relevance: { pagerank: 10 } };
const date = new Date();

const props = {
  user,
  date,
  relevance: false,
  setSelected: jest.fn()
};

test('Snapshot AvatarBox Web', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <AvatarBoxWeb {...props} />
    </MemoryRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
