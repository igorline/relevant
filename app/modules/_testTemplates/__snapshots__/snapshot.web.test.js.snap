// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot AvatarBox Web 1`] = `
.c0 {
  color: #999999;
  font-size: 10px;
  font-family: Arial,sans-serif;
}

.c0 a {
  color: #999999;
}

<div
  className="avatarBox "
>
  <a
    className="avatar"
    href="/user/profile/handle"
    onClick={[Function]}
    style={
      Object {
        "backgroundImage": "url(/img/default_user.jpg)",
        "height": undefined,
        "width": undefined,
      }
    }
  >
    Name
  </a>
  <div
    className="userBox"
  >
    <div
      className="bebasRegular username"
    >
      <a
        href="/user/profile/handle"
        onClick={[Function]}
      >
        Name
      </a>
      <span>
        <span
          alt="R"
          className="r"
          style={
            Object {
              "backgroundImage": "url(/img/r-emoji.png)",
            }
          }
        />
        10
      </span>
    </div>
    <div
      className="c0"
    >
      <a
        href="/user/profile/handle"
        onClick={[Function]}
      >
        @
        handle
      </a>
      A Few Seconds Ago
    </div>
  </div>
</div>
`;
